javascript/operators/conditional.json

```diff
@@ -4,7 +4,6 @@
       "conditional": {
         "__compat": {
           "description": "Conditional operator (<code>c ? t : f</code>)",
-          "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Operators/Conditional_Operator",
```

Comment:
This differs from the real URL (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator) by capitalization only